operational/flexera/custom_branding/README.md

```diff
@@ -0,0 +1,74 @@
+# Configure Custom Branding
+
+## What It Does
+
+This policy template allows a customer to apply their own branding to the FlexeraOne platform quickly and consistently.
+
+## How It Works
+
+- Apply the custom branding policy template in the tenant ensuring you select the credentials required.
+- For non MSP Customers you can only apply the All Customer Settings Category.
+- For MSP Customers you can apply the All Customer Settings Category and the MSP Specific Settings Category.
+- Select the areas that require branding changes by either selecting Yes/No for show/hide features, or, entering a URL to Images or Links or text to replace existing text.
+- If there are items that do not need to be changed, leave the defaults of either No or blank and it will be ignored.
+- Apply the policy template and review if there are any errors, an incident will be generated if an item fails.
+- After the policy template has been applied and changes are in effect, it is recommended to terminate the applied policy.
+
+## Input Parameters
+
+This policy template has the following input parameters required when being applied.
+
+### Applies to whole policy (Optional)
+
+- *Email Addresses* - A list of email addresses to notify
+
+### Category : All Customer Settings
+
+- *Nav Logo Image* - The URL of the image file to use in place of the Flexera logo that displays in the app navigation’s expanded state. Must start with http : // or https : //
+- *Nav Logo Small image* - The URL of the image file to use in place of the small Flexera logo that displays in the app navigation’s collapsed state. Must start with http : // or https : //
+- *Marketing Footer Hide* - Yes or No to show/hide the marketing footer on the landing page. (Default is No)
+- *Marketing Submit Ideas Hide* - Yes or No to show/hide the Submit Ideas link on the landing page. (Default is No)
+- *Marketing Support Plan Hide* - Yes or No to show/hide the Support Plan Information link on the landing page. (Default is No)
+- *Marketing Open New Case Link* - A URL to be used for the Open New Case link on the landing page. Must start with http : // or https : //

```

Comment:
You should escape characters or use backticks rather than adding spaces to `http://`